frontend/app/components/uikit/menu-button/menu-button.scss

```diff
@@ -29,4 +29,18 @@
         @apply outline-transparent;
     }
   }
+
+  &.dark {
```

Comment:
dont use .dark class here instead use &--dark to corespond to BEM naming standards